### DISCUSSION TITLE
Where settings are stored ?

### DISCUSSION THREAD

**Discussion by EdiDD** [ddterm/gnome-shell-extension-ddterm#97] [Q&A] [answered]
After remove old extension and installing a new one all my settings where untouched. Where ddterm config is stored ? I can't find anything in gsettings and all files were removed after uninstall.

**Comment by amezin** [comment, marked as answer]
https://github.com/amezin/gnome-shell-extension-ddterm/blob/e7bbfeebe3c1e32f6a83e1724075ecb1045e2b2b/schemas/com.github.amezin.ddterm.gschema.xml#L141

**Comment by coc0nut** [comment]
Hi there... looking for this path.. trying gsettings set com.github.amez.ddterm window-position bottom but says no such schema..

Nice terminal btw! ;)
